src/mumble/GlobalShortcut_win.cpp

```diff
@@ -417,7 +440,7 @@ void GlobalShortcutWin::injectRawInputMessage(HRAWINPUT handle) {
 	switch (input->header.dwType) {
 		case RIM_TYPEMOUSE: {
 			const RAWMOUSE &mouse = input->data.mouse;
-			emit mouseMessage(mouse.usButtonFlags, mouse.usButtonData);
+			m_msgQueue.emplace(new MsgMouse(mouse.usButtonFlags));
```

Comment:
`std::make_unique` should generally be preferred to instantiate a `unique_ptr`

Reply:
Yes, but it would require to specify the type (`MsgRaw` in this case).

Reply:
Why is this a problem? :thinking: